grafana/grafana
# InfluxDB: Fix applying ad-hoc filters when querying with backend mode

**What is this feature?**

When `influxdbBackendMigration` is enabled we do not apply the ad-hoc filters to the query. This PR is fixing that bug. 

**Why do we need this feature?**

To be able to use ad-hoc filters with backend mode enabled.

**Who is this feature for?**

InfluxDB backend mode users

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/grafana/issues/74972

**Special notes for your reviewer:**

### How to test
- Enable `influxdbBackendMigration` feature toggle
- Create a simple dashboard with an ad-hoc variable (datasource influxdb)
- Have an ad-hoc filter selected.
- Create a simple query
- Observe with a proxy (i.e. [mitmproxy](https://mitmproxy.org/)) that ad-hoc filters are not applied to the query
- Switch to this branch and retry. 

Please check that:
